iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/HalfLineUtil.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.util;
+
+import org.apache.commons.math3.complex.Complex;
+
+import com.powsybl.iidm.network.Branch;
+import com.powsybl.iidm.network.TieLine;
+import com.powsybl.iidm.network.util.LinkData.BranchAdmittanceMatrix;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+
+public final class HalfLineUtil {
```

Comment:
I am not sure the name of this class is relevant. Is it not more related to tie line?

Reply:
Yes, better TieLineUtil as you implemented in the commit.